src/dist/replication/lib/replica_2pc.cpp

```diff
@@ -74,6 +74,25 @@ void replica::on_client_write(dsn::message_ex *request, bool ignore_throttling)
         return;
     }
 
+    if (_is_bulk_load_ingestion) {
+        // reject write requests during ingestion
+        response_client_write(request, ERR_BUSY);
```

Comment:
Will ERR_BUSY be ambiguous with the rate-limit error of write? How should the client handle this error?

Reply:
It is for compatibility. If I return a new error code like 'ERR_INGESTING', old version client won't recognize it. So I use `ERR_BUSY`, current client version know it.

Reply:
I think you can use `ERR_IO_PENDING` or someone else. Because `ERR_BUSY` is a common error code, which means rate limit error.

Reply:
I use `ERR_OPERATION_DISABLED` instead, this error code may be more suitable.